kedro_airflow_k8s/operators/node_pod.py

```diff
@@ -37,6 +37,8 @@ def __init__(  # pylint: disable=too-many-arguments
         limits_cpu: Optional[str] = None,
         limits_memory: Optional[str] = None,
         node_selector_labels: Optional[Dict[str, str]] = None,
+        labels: Optional[Dict[str, str]] = None,
+        tolerations: Optional[List[Dict[str, str]]] = None,
```

Comment:
These parameters are missing in function doc section.

Reply:
fixed